File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.cs
using System;

public class Solution {
    public int[] solution(string my_string) {
        int[] answer = new int[52];
        for(int i = 0 ; i < my_string.Length;i++)
        {
            char num = my_string[i];
            if(65 <= num && num <= 90)
            {
                answer[num - 65] += 1;
            }
            else if(97 <= num && num <= 122)
            {
                answer[num - 97 + 26] += 1;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    answer.reserve(52);
    
    for(int i = 0; i < 52; i++)
        answer.push_back(0);
    
    for(int i = 0; i < my_string.size(); i++)
    {
        char ch = my_string[i];
        if(65 <= ch && ch <= 95)
            answer[ch - 65] += 1;
        else if(97 <= ch && ch <= 122)
            answer[ch - 97 + 26] += 1;
    }
    return answer;
}